src/components/App.tsx

```diff
@@ -1,14 +1,13 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import { hot } from 'react-hot-loader/root';
 import { Provider } from 'react-redux';
-import { info } from 'electron-log';
 import { ConnectedRouter } from 'connected-react-router';
 import { StoreProvider } from 'easy-peasy';
 import store, { hashHistory } from 'store';
 import { Routes } from 'components/routing';
+import DockerContainerShutdown from './appExit/DockerContainerShutdown';
 
 const App: React.FC = () => {
-  useEffect(() => info('Rendering App component'), []);
```

Comment:
Why do you remove this log statment here?

Reply:
My bad. This is not intentional. I will put it back.